src/components/Form/FormInput.react.js

```diff
@@ -37,7 +49,15 @@ function FormInput({
   );
   return !icon ? (
     <React.Fragment>
-      <input className={classes} />
+      <input
+        type={type}
+        className={classes}
+        placeholder={placeholder}
+        name={name}
+        value={value}
+        disabled={disabled}
```

Comment:
What is the difference between disabled and readOnly?

Reply:
They look and act the same for users but disabled inputs don't get sent on form submit, readOnly do